applications/luci-app-statistics/root/etc/init.d/luci_statistics

```diff
@@ -1,8 +1,86 @@
 #!/bin/sh /etc/rc.common
 
+# run luci_statistics before collectd starts (80) and stop after
+# collectd stops (10):
 START=79
+STOP=11
 USE_PROCD=1
 
+# We only want to restore a sysupgrade backup file if
```

Comment:
Might want to add a note about this being better than a cronjob, as it doesn't constantly write to flash on devices with non-durable storage.

Reply:
I added something, let me know if you like it.